common/logisticspipes/api/IProgressProvider.kt

```diff
@@ -0,0 +1,15 @@
+package logisticspipes.api;
+
+/**
+ * Implement this in a TileEntity.
+ *
+ * This will allow LP to access the progress of a TileEntity connected
+ * to a Crafting Pipe and display it inside the pipe's crafting tree view.
+ */
+interface IProgressProvider {
+	/**
+	 * TODO convert to Float and use range 0.0f to 1.0f as that is how I have seen it done everywhere?
+	 * @return a value between 0 and 100 as completion percentage
+	 */
+	fun getProgress(): Byte
```

Comment:
Is it a good idea to have a name that is this common? The idea behind naming it that way, was to avoid collisions with other mods when they try to implement it.

Reply:
Ok, then isEnabled in IHudArmor is also common. A convention should be established for the functions in this folder. A suffix "LP" maybe?